### ISSUE TITLE
Application crashes on first launch

### ISSUE THREAD

**Issue by valentin-kaiser** [AntonSkrub/GoGit-Integration#8]
```
PS C:\Workspace\GoGit-Integration\cmd> go run .\main.go
time="2023-01-13T10:56:14+01:00" level=info msg="Grabbing configuration..."
time="2023-01-13T10:56:14+01:00" level=panic msg="config file not found: CreateFile ./config/config.yml: Das System kann den angegebenen Pfad nicht finden."
panic: (*logrus.Entry) 0xc0000b6000
```

**Comment by valentin-kaiser** [comment]
It would be practical to have a default configuration

**Comment by AntonSkrub** [comment]
Fixed it in commit c16b934
